src/modules/person/api/lock-user.body.params.ts

```diff
@@ -0,0 +1,18 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsBoolean, IsString } from 'class-validator';
+
+export class LockUserBodyParams {
+    @IsBoolean()
+    @ApiProperty({
+        required: true,
+        nullable: false,
+    })
+    public readonly lock!: boolean;
+
+    @IsString()
+    @ApiProperty({
+        required: true,
+        nullable: false,
+    })
+    public readonly locked_from!: string;
```

Comment:
Maybe also include a "@IsNotEmpty()", just in case

Reply:
Done!